values.yaml

```diff
@@ -513,6 +554,14 @@ connectInject:
   # method for Connect inject, set this to the name of your auth method.
   overrideAuthMethodName: ""
 
+  # aclInjectToken refers to a Kubernetes secret that you have created that contains
```

Comment:
is this required for namespaces? Seems like a good idea, just wondering why it's in this PR.

Reply:
This is required for namespaces. Previously, the connect injector didn't need an ACL token as all of the operations that required it for that process were done in the init container during injection with an acl provided by the auth method. With namespaces, however, the connect injector may need to create a Consul namespace which requires privileges if acls are enabled. This allows folks to provide that token without having run the bootstrapping, as is also available as an option for sync catalog.